projects/cdk/utils/miscellaneous/create-options.ts

```diff
@@ -0,0 +1,7 @@
+import {InjectionToken} from '@angular/core';
+
+export function tuiCreateOptions<T>(defaults: T): InjectionToken<T> {
+    return new InjectionToken<T>(``, {
```

Comment:
`` - is it good?

Reply:
I'm not sure it is of any help since it is mostly shown when there is no provider with this name, but all those tokens have default values so they never cause this situation really.